src/OpenyActivityFinderSolrBackend.php

```diff
@@ -323,7 +323,7 @@ public function processResults(ResultSet $results, $log_id) {
         continue;
       }
       $fields = $result_item->getFields();
-      $dates = $entity->field_session_time->referencedEntities();
+      $dates = $entity->field_session_time ? $entity->field_session_time->referencedEntities() : [];
```

Comment:
@podarok I believe `$entity->field_session_time` should give you an object of FieldItemList class... could you please change it to `$entity->field_session_time->isEmpty()`?

Reply:
@duozersk it's null for some reason for Francis https://github.com/ymcatwincities/openy/issues/2388#issuecomment-769979955